tests/platform_tests/sfp/test_sfputil.py

```diff
@@ -390,6 +391,26 @@ def test_check_sfputil_eeprom(duthosts, enum_rand_one_per_hwsku_frontend_hostnam
             assert parsed_eeprom[intf] == "SFP EEPROM detected"
 
 
+@pytest.mark.device_type('physical')
+def test_check_sfputil_eeprom_hexdump(duthosts, enum_rand_one_per_hwsku_frontend_hostname,
+                                      enum_frontend_asic_index, conn_graph_facts, xcvr_skip_list):
+    """
+    @summary: Check eeprom hexdump using 'sfputil show eeprom-hexdump'
+    """
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    global ans_host
+    ans_host = duthost
+    portmap, dev_conn = get_dev_conn(duthost, conn_graph_facts, enum_frontend_asic_index)
+
+    logging.info("Check output of '{}'".format(cmd_sfp_eeprom_hexdump))
+    sfp_eeprom_hexdump = duthost.command(cmd_sfp_eeprom_hexdump)
+    parsed_eeprom_hexdump = parse_eeprom_hexdump(sfp_eeprom_hexdump["stdout_lines"])
+    for intf in dev_conn:
+        if intf not in xcvr_skip_list[duthost.hostname]:
+            assert intf in parsed_eeprom_hexdump, "Interface is not in output of 'sfputil show eeprom-hexdump"
```

Comment:
This error message is unclear, which interface is missing? can we print the interface name?
Same goes for the next assert, no error message at all